src/main/io/serial_msp.c

```diff
@@ -1020,14 +1021,18 @@ static bool processOutCommand(uint8_t cmdMSP)
         break;
 
     case MSP_RX_CONFIG:
-        headSerialReply(12);
+        headSerialReply(17);
         serialize8(masterConfig.rxConfig.serialrx_provider);
         serialize16(masterConfig.rxConfig.maxcheck);
         serialize16(masterConfig.rxConfig.midrc);
         serialize16(masterConfig.rxConfig.mincheck);
         serialize8(masterConfig.rxConfig.spektrum_sat_bind);
         serialize16(masterConfig.rxConfig.rx_min_usec);
         serialize16(masterConfig.rxConfig.rx_max_usec);
+        serialize8(masterConfig.rxConfig.rcSmoothing);
+        serialize8(masterConfig.rxConfig.rcSmoothInterval);
+        serialize16(masterConfig.rxConfig.airModeActivateThreshold);
+        serialize8(masterConfig.rxConfig.rcSmoothInterval);
```

Comment:
@martinbudden 
Cheers. Fixed and rebased. 

I guess only configurator work needs to be done now before merging. I cant see anything wrong with it